src/plugins/mini/Examples/read.ini

```diff
@@ -0,0 +1,15 @@
+keyWithoutLeadingWhitespace = 	valueWithLeadingWhiteSpace
```

Comment:
Please do not name the folder `Examples` but `mini` (like the plugin is named). Otherwise we have confusing directories or even get a clash when installing these files.